lib/xboxrt/libc_extensions/wchar.h

```diff
@@ -1,8 +1,11 @@
 #ifndef _XBOXRT_WCHAR_H
 #define _XBOXRT_WCHAR_H
 
+#include <assert.h>
```

Comment:
Should move to source file - shouldn't be needed in header

Reply:
Done.